frontend/src/App.css

```diff
@@ -12,6 +12,12 @@
     margin: 0;
   }
 
+  .App {
+    position: relative;
+    min-height: 100vh;
+    padding-bottom: 10rem;
+  }
```

Comment:
`.App` has different naming conventions (casing) than the rest of the css

Reply:
Yes, I thought this was odd but decided not to change it as that was the class already given to the top-level div. On further inspection I can't see anything else that refers to `.App` though, so I will rename to `app`.